pkgs/development/python-modules/pytest-asyncio/default.nix

```
@@ -27,6 +29,9 @@ buildPythonPackage rec {
   build-system = [ setuptools-scm ];
 
   buildInputs = [ pytest ];
+  dependencies = lib.optional (!python.pythonAtLeast "3.11") [
```

```suggestion
  dependencies = [
```
since it's `null` otherwise
Also, there is `pythonOlder` and no need for `python.`

---

Oops, I forgot that we already discussed this and said that it would lead to a high rebuild count. So please just use `pythonOlder`.